src/__tests__/standardMethods.spec.ts

```diff
@@ -133,4 +160,58 @@ describe('standard methods', () => {
       }
     });
   });
+  describe('generateDeleteEndpoint', () => {
+    test('automatic chunking for delete', async () => {
+      const metadataMap = new MetadataMap();
+      const ids = new Array(4500).fill(null).map((_, index) => ({
+        id: Number(index),
+      }));
+      const remove = generateDeleteEndpoint(
+        axiosInstance,
+        '/path',
+        metadataMap
+      );
+      const chunkCounter = countChunks(axiosMock, 'path/delete');
+      const response = await remove(ids);
+      expect(chunkCounter.value).toBe(5);
+      expect(response).toMatchObject({});
+      expect(metadataMap.get(response)).toBeDefined();
+    });
+  });
+  describe('generateUpdateEndpoint', () => {
+    test('automatic chunking for update', async () => {
+      const metadataMap = new MetadataMap();
+      const changes = new Array(2003).fill(null).map((_, index) => ({
+        id: Number(index),
+      }));
+      const update = generateUpdateEndpoint(
+        axiosInstance,
+        '/path',
+        metadataMap
+      );
+      const chunkCounter = countChunks(axiosMock, 'path/update');
+      const response = await update(changes);
+      expect(chunkCounter.value).toBe(3);
+      expect(response.length).toBe(changes.length);
+      expect(metadataMap.get(response)).toBeDefined();
+    });
+  });
+  describe('generateInsertEndpoint', () => {
+    test('automatic chunking for insert', async () => {
+      const metadataMap = new MetadataMap();
```

Comment:
Can we refactor all of this? These lines are common for all the tests to reduce number of duplicate lines

Reply:
@eirikdahlen Still a lot of similar code left